test/unit/test_loss_image.py

```diff
@@ -31,6 +31,23 @@ def test_lncc(self):
         expected = [-1, -1]
         assert is_equal_tf(got, expected)
 
+        got = image.dissimilarity_fn(
```

Comment:
emm sorry, seems like I didn't write the test very well the last time I changed it.

Could we use `parameterize` for the tests? I can do an example in `test_ssd(self)` ;)